app/Repositories/Admin/User/UserRepository.php

```diff
@@ -61,7 +59,8 @@ public function create($data)
             $data['remember_token'] = Str::random(64);
             $data['email_verified_at'] = now();
             if ($user = $this->model->create($data)) {
-                // $this->dispatch((new AssignStarterProjects($user))); For now - not needed
+                AssignStarterProjects::dispatch($user, $user->createToken('auth_token')->accessToken)->delay(now()->addSecond())->onQueue('starterProjects');
```

Comment:
Same here for token generation.

Reply:
Same above case for cloning... I don't have any choice here except to send the token.

Reply:
Same here.

Reply:
Have explained this in the above conversation.